.env.sample

```diff
@@ -6,4 +6,11 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
 GOOGLE_ANALYTICS_4_GTM_CONTAINER_ID="GTM-KD86CMZ"
 GA4_ENABLED="true"
 ANALYTICS_DOMAIN="localhost"
-LOG_LEVEL="debug"
\ No newline at end of file
+LOG_LEVEL="debug"
+
+# QA Env
+IPV_STUB_URL=
+IPV_BASE_URL=
```

Comment:
What is the difference between these 2 vars?

Reply:
IPV_STUB_URL is the URL used to start the CIC stub service (https://f2f-ipv-stub-ipvstub.review-o.dev.account.gov.uk/start)

IPV_BASE_URL is the baseURL for the F2F FE (https://f2f-cri-front.review-o.dev.account.gov.uk)